cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -684,7 +684,7 @@ func setField(rv reflect.Value, path []string, v reflect.Value) error {
 								},
 							}))
 						}
-					case ft.Elem().Kind() == reflect.Int32 && vt.Elem().Kind() == reflect.String:
+					case vt.Elem().Kind() == reflect.String:
 						if reflect.PtrTo(ft.Elem()).Implements(textUnmarshalerType) {
```

Comment:
Would be even nicer to pull that `if` into the `case`. And I guess we can then do the same on L526.

Reply:
Done. The reason I didn't do this because we lose the `does not implement encoding.TextUnmarshaler` error and instead receive the default `is not convertible to` error.